README.md

```diff
@@ -2,6 +2,7 @@
 The iRODS indexing capability provides a policy framework around both full text and metadata indexing for the purposes of enhanced data discovery.  Logical collections are annotated with metadata which indicates that any data objects or nested collections of data object should be indexed given a particular indexing technology, index type and index name.
 
 # Configuration
+
 ### Collection Metadata
```

Comment:
this skip from one # to three ### seems incorrect.

Reply:
Agreed. Updated.